### ISSUE TITLE
Better support for Sandstorm

### ISSUE THREAD

**Issue by arjunv** [cryptag/cryptag#101]
One of the best usecases of your awesome tool must be the `Sandstorm` integration.
With `Sandstorm` appearing to have a much awaited revival, I think it's time to iron out some wrinkles and put some makeup on.

- Add an option for deleting entries from command line
 Right now, it just says "PlainTag `delete` not found"
- Prettify output
 Tags and credentials would be sufficient by default. id, app, type or created date can be shown with a `-v` flag
- A web interface in Sandstorm to manage the credentials
- A reminder in the docs for folks to use methods for preventing their credentials being logged the shell history
  for eg, enabling `set hist_ignore_space` in ZSH along with an alias like `alias cpass=" cpass"`

**Comment by elimisteve** [comment]
Hi @arjunv , thanks for your interest!

I use `cpass` as my password manager to this day :-). There is a `delete` subcommand on my version but maybe I never pushed it to GitHub? Or maybe `cryptag delete` is what's missing?

`alias cpass=" cpass"` is clever; thanks for the tip!

**Comment by arjunv** [comment]
Yup. the `delete` subcommand is missing in the version in github.
